docs/logger.rst

```diff
@@ -118,30 +118,38 @@ You can use predefined celery task ``security.tasks.call_django_command`` to run
 
   There are several methods of ``LoggedTask`` which you can use for your advanced tasks logic.
 
-  .. property:: task_log
+  .. property:: task_run_log
 
-    This method returns an instance of CeleryTaskLog related to your task.
+    This method returns an instance of CeleryTaskRunLog related to your task.
 
   .. method:: on_apply_task(task_log, args, kwargs, options)
 
     This method is called before the task is queued. You can override this method.
 
-  .. method:: on_start_task(task_log, args, kwargs)
+  .. method:: on_start_task(task_run_log, args, kwargs)
 
     This method method is called when the task was started.
 
-  .. method:: on_success_task(task_log, args, kwargs, retval)
+  .. method:: on_success_task(task_run_log, args, kwargs, retval)
 
     This method method is called when the task was successfully completed.
 
-  .. method:: on_failure_task(task_log, args, kwargs, exc)
+  .. method:: on_failure_task(task_run_log, args, kwargs, exc)
 
     This method method is called when the task raised an exception and is not retried.
 
-  .. method:: on_retry_task(task_log, args, kwargs, exc)
+  .. method:: on_retry_task(task_run_log, args, kwargs, exc)
 
     This method method is called when the task raised an exception and is retried.
 
+  .. method:: apply_async_on_commit(args=None, kwargs=None, **options)
+
+    This method has same behaviour like ``apply_async`` but run task is with ``on_commit`` django signal. Therefore is inicialized in the end of django atomic block.
```

Comment:
This method has the same behaviour as ``apply_async`` but runs task with ``on_commit`` django signal. Therefore it is initialized at the end of the django atomic block.